common/src/androidTest/java/com/microsoft/identity/common/adal/internal/util/HashMapExtensionTests.java

```diff
@@ -61,7 +61,7 @@ public void testURLFormDecodeNegative() throws IllegalArgumentException,
         final String methodName = "urlFormDecode";
         Object object = ReflectionUtils.getNonPublicInstance("com.microsoft.identity.common.adal.internal.util.HashMapExtensions");
```

Comment:
This method is public now I think.

Reply:
Great - thanks. I'll open up a work item and we can apply this change forwards

Reply:
Work item is here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/272

Thanks @shoatman